### PR TITLE
refactor(docs): read the description

### PR DESCRIPTION
**Pull request by danielsogl**
- Added missing platforms
- Unified GitHub links
- Added missing installVariables

### CODE REVIEW

File: src/@ionic-native/plugins/base64-to-gallery/index.ts [hunk 1]
```diff
@@ -23,7 +23,7 @@ import { Cordova, Plugin, IonicNativePlugin } from '@ionic-native/core';
 @Plugin({
   pluginName: 'Base64ToGallery',
   plugin: 'cordova-base64-to-gallery',
-  pluginRef: 'cordova',
+  pluginRef: 'cordova.base64ToGallery',
```
Comment (ihadeed) [inline src/@ionic-native/plugins/base64-to-gallery/index.ts]: This should stay as `cordova` only. 

`cordova.base64ToGallery` is a function, not an object. Our wrapper wouldn't work unless if the pluginRef is set to `cordova`.

File: src/@ionic-native/plugins/appodeal/index.ts [hunk 2]
```diff
@@ -22,9 +22,9 @@ import { Injectable } from '@angular/core';
  */
 @Plugin({
   pluginName: 'Appodeal',
-  plugin: 'https://github.com/appodeal/appodeal-cordova-plugin',
+  plugin: 'https://github.com/appodeal/appodeal-cordova-plugin.git',
```
Comment (ihadeed) [inline src/@ionic-native/plugins/appodeal/index.ts]: Any specific reason  for adding the `.git` prefix here (and other occurrences below) ?

Comment (danielsogl) [inline src/@ionic-native/plugins/appodeal/index.ts]: I want to unify the plugins if they download the files from GitHub.
If you try to download the files without .git, GitHub redirects to the .git file.

File: src/@ionic-native/plugins/base64-to-gallery/index.ts (hunk 1 above)
Comment (danielsogl) [inline src/@ionic-native/plugins/base64-to-gallery/index.ts]: I will fix that

File: src/@ionic-native/plugins/appodeal/index.ts (hunk 2 above)
Comment (danielsogl) [inline src/@ionic-native/plugins/appodeal/index.ts]: And if you open an .git link, GitHub redirects to the repo page without .git

Comment (danielsogl) [inline src/@ionic-native/plugins/appodeal/index.ts]: And the official Cordova docs also linking to the .git links.

**Pull request merged**